LICENSE

```diff
@@ -1,6 +1,13 @@
-Float128 is licensed under the MIT License:
+Quadmath.jl is licensed under the MIT License:
 
-Copyright (c) 2016: Harald Hofstätter, Simon Byrne
+Copyright (c) 2016-2019: Harald Hofstätter, Simon Byrne
```

Comment:
You should add yourself!

Reply:
Happily, your `ccall` macro replaces the only really original stuff from me!

Reply:
I still think there is sufficient contribution to justify it.